app/views/manage_outcomes/outcomes/_outcomes.html.erb

```diff
@@ -10,7 +10,7 @@
     <% course.outcomes.each do |outcome| %>
       <tr>
         <td><%= outcome %></td>
-        <td><%= outcome.assessments_count %></td>
+        <td><%= render "progress_bar", amount: outcome.active_assessments_count, total: outcome.active_assessments_count %></td>
```

Comment:
@derekprior I saw that you merged some of the changes we needed to get the progress bar to work. I wasn't sure how to get active assessments count with results or if that is still something you need to do.